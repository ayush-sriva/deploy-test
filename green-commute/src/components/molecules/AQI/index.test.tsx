/**
 * @author Asish Kalintha <asish.sugun@zemosolabs.com>
 */
import renderer from 'react-test-renderer';
import AQIIndex from '.';

describe('AQIIndex', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<AQIIndex children='descreption' index='900' />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
