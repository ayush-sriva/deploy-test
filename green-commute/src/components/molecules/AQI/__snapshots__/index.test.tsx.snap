// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AQIIndex renders correctly 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-wrap-xs-nowrap makeStyles-container-1 css-19kb62l-MuiGrid-root"
>
  <div
    className="MuiGrid-root MuiGrid-item makeStyles-aqi-2 css-13i4rnv-MuiGrid-root"
  >
    <span
      className="MuiTypography-root MuiTypography-body3 makeStyles-innerText-3 css-37jdci-MuiTypography-root"
    >
      900
    </span>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item makeStyles-text-4 css-13i4rnv-MuiGrid-root"
  >
    <h2
      className="MuiTypography-root MuiTypography-h2 css-1sra7t5-MuiTypography-root"
    >
      descreption
    </h2>
  </div>
</div>
`;
